startupscript/remount-on-restart.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# remount-on-restart.sh
+#
+# Remounts buckets for the logged in user when a devcontainer instance is restarted.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+if [[ $# -ne 4 ]]; then
```

Comment:
Would be preferable for the 4th arg to be more of an enum like "login/nologin" and the downstream variable be more descriptive of what is being logged into (cloud, workbench, github).
I realize that's a more intrusive change than just fixing what you are currently working on, but would be a good sweep to make at some point.

Reply:
Agreed!  [BENCH-2566](https://verily.atlassian.net/browse/BENCH-2566) captures removing all the `login`-conditional code, at which point we'll remove the arg entirely.